docs/reference/mapping/params/position-increment-gap.asciidoc

```diff
@@ -22,7 +22,9 @@ GET /my_index/groups/_search
 {
     "query": {
         "match_phrase": {
-            "names": "Abraham Lincoln" <1>
```

Comment:
Thanks Nik, actually the problem should happen for the second example 
```
GET /my_index/groups/_search
{
    "query": {
        "match_phrase": {
            "names":  "Abraham Lincoln",
            "slop": 101 (2)
        }
    }
}
```

It gives me 

```

{
   "error": {
      "root_cause": [
         {
            "type": "parsing_exception",
            "reason": "[match] query parsed in simplified form, with direct field name, but included more options than just the field name, possibly use its 'options' form, with 'query' element?",
            "index": "my_index",
            "line": 4,
            "col": 39
         }
      ],
....
```
I changed the first example here to match it, showing the `slop` is the only thing we are changing in the example. Sorry for the confusing.